db/migrate/20211109200705_backfill_type_on_vita_partner.rb

```diff
@@ -0,0 +1,16 @@
+class BackfillTypeOnVitaPartner < ActiveRecord::Migration[6.0]
+
+  def up
+    ActiveRecord::Base.connection.execute(
+      "UPDATE vita_partners SET type='Organization' WHERE parent_organization_id IS NULL"
+    )
+
+    ActiveRecord::Base.connection.execute(
+      "UPDATE vita_partners SET type='Site' WHERE parent_organization_id IS NOT NULL"
+    )
+  end
+
+  def down
+    VitaPartner.all.update_all(type: nil)
```

Comment:
Ho ho ho, this could be raw SQL and that'd be better, except we don't use `down` in prod anyway, so it's no problem to leave it this way.